collections/_version2-english/9-fork/4-migration-of-the-objects.md

```diff
@@ -103,15 +107,38 @@ The summary is shown in the following figure:
+
+The migration of the non-forking markets is not mandatory. Even if a fork ends, you can still trade and [settle]({{glossary_settlement}}) your [shares]({{glossary_share}}) on the markets in the parent universe (See [restrictons on use]({{url_restrictions_on_use}}) for details). However, after a fork starts, markets in the parent universe cannot be [finalized]({{glossary_finalized_market}}). For the market to be finalized, you need to migrate it to the winning universe.
+
+Although the person who migrates a non-forking market needs to do somethings (such as supply creation bond, submit the initial report, and pay the [transaction fee]({{glossary_transaction_fee}}) for that), they can be a market creator of the market in the winning universe, so they will get [creator fees]({{glossary_creator_fee}}) paid by traders on the market in the winning universe. It might motivate migrating markets.
+
 ## Staked REP
+When a [fork]({{glossary_fork}}) starts, all staked [REP]({{glossary_reputation_token}}) on the non-forking markets are un-staked and refunded to REP holders who staked it so that they migrate their REP during the [forking period]({{glossary_forking_period}}).
+
 ## Attached Objects
+The objects which are attached to the non-forking market are:
+ - [share]({{glossary_share}})
+ - [unfilled order]({{glossary_unfilled_order}})
+ - [creator fee]({{glossary_creator_fee}})
+ - [validity bond]({{glossary_validity_bond}})
+
+Those objects go where the market goes, they cannot be migrated separately.
+
+[Creation bond]({{glossary_creation_bond}}), which is paid by the [market creator]({{glossary_market_creator}}) when creating the market in the [parent universe]({{glossary_parent_universe}}), is refunded to either the market creator or the [initial reporter]({{glossary_initial_reporter}}). At the time of starting a [fork]({{glossary_fork}}), if the market has not gotten the [initial reporting]({{glossary_initial_report}}), this bond is refunded to the market creator. If already gets the initial reporting, refunded to the [initial reporter]({{glossary_initial_report}}). Because "gotten the initial reporting" means that the creation bond is already used by the initial reporter as the initial reporter's stake.
```

Comment:
```suggestion
[Creation bond]({{glossary_creation_bond}}), which is paid by the [market creator]({{glossary_market_creator}}) when creating the market in the [parent universe]({{glossary_parent_universe}}), is refunded to either the market creator or the [initial reporter]({{glossary_initial_reporter}}). At the time of starting a [fork]({{glossary_fork}}), if the market has not received an [initial reporting]({{glossary_initial_report}}), this bond is refunded to the market creator. If it already has received the initial report, then it refunds the [initial reporter]({{glossary_initial_report}}). Because "received the initial reporting" means that the creation bond is already used by the initial reporter as the initial reporter's stake.
```